Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java

```diff
@@ -934,6 +936,8 @@ public void bulkInsertArtifacts() throws EamDbException {
                 if (bulkArtifactsCount == 0) {
                     return;
                 }
+                
+                TimingMetric timingMetric = EnterpriseHealthMonitor.getTimingMetric("Correlation Engine: Bulk insert");
```

Comment:
I think the "Correlation Engine: Bulk insert" string should be declared a constant, given that we are using it as an identifier to start and stop the timer.

Reply:
It's only used in creating the timer - stopping is done on the metric object so we don't need to match it up with anything. I don't think making it a constant gains us much since it will only be used in a single spot.